src/hierarchies/priors/nig_prior_model.h

```diff
@@ -11,6 +11,13 @@
 #include "hyperparams.h"
 #include "src/utils/rng.h"
 
+//! A conjugate prior model for the univariate normal likelihood, that is
+//!     mu | var ~ N(mu0, var / lambda)
+//!     var ~ IG(a,b)
+//! With several possibilies for hyper-priors on mu and var. We have considered
+//! a normal prior for mu0 and a NGG for (mu0, a, b) in addition to fixing
```

Comment:
We should write Normal-Gamma-Gamma in full in addition to its acronym

Reply:
Agree, thanks!